modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -1132,6 +1132,16 @@ private void _validateBusinessTypeEncrypted(
 				"Business type encrypted is disabled");
 		}
 
+		if (Validator.isNull(PropsValues.OBJECT_ENCRYPTION_ALGORITHM)) {
+			throw new ObjectFieldBusinessTypeException(
+				"Encryption algorithm is required");
```

Comment:
Once the ObjectFieldBusinessTypeException is generic, it would be nice to have the business type included in the error message, like [here](https://github.com/liferay/liferay-portal/blob/1ced75d3265746da90dec6e1f6060faa17a39fc5/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java#L1131), [here](https://github.com/liferay/liferay-portal/blob/1ced75d3265746da90dec6e1f6060faa17a39fc5/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java#L1139) and [here](https://github.com/liferay/liferay-portal/blob/1ced75d3265746da90dec6e1f6060faa17a39fc5/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java#L1200).

Reply:
Suggestion: Encryption algorithm is required for business type encrypted